packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,9 @@
 <template>
   <template if:true={shouldDisplay}>
-    <li class={tabClass} title={label} role="presentation">
-      <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
-    </li>
+    <div tabindex="0" role="tab" onkeydown={onKeyDown}>
+      <li class={tabClass} title={label} role="presentation">
+        <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
+      </li>
+    </div>
```

Comment:
```suggestion
    <li class={tabClass} title={label} role="presentation">
      <button class="slds-button slds-tabs_default__link" role="tab" onclick={select}>{label}</button>
    </li>
```

Reply:
Make sure to have something so the button doesn't have a huge `line-height` look in my last PRs to set it correctly.